tests/cli/test_cli.py

```diff
@@ -569,3 +569,16 @@ def test_main_py_log_level_debug(self):
         exit_status = proc.returncode
         assert "DEBUG" in out_str
         assert exit_status == 0
+
+    def test_num_iter(self):
+        setup()
+        with TestCase.assertLogs("peekingduck.cli.logger") as captured:
+            result = CliRunner().invoke(cli, ["run", "--num_iter", 50])
+            assert (
+                captured.records[0].getMessage()
+                == "Successfully loaded run_config file."
+            )
+            assert captured.records[1].getMessage() == init_msg(PKD_NODE)
+            assert captured.records[2].getMessage() == init_msg(PKD_NODE_2)
+            assert captured.records[-1].getMessage() == "Run pipeline for 50 iterations"
```

Comment:
would it be better to assign 50 to a variable and then use it in the expected logging message so it's clearer how these values are related?

Reply:
Refactored as suggested.